### PR TITLE
Seances implementation

### PR DESCRIPTION
**Pull request by meef3k**
- Created models and migrations for Movies, Halls and Seances
- Created controllers for Seances and Movies
- Created views for Seances and Movies
- Added routes
- Added validations for models
- Changed rows in Seances

### CODE REVIEW

File: app/controllers/seances_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  end
+  # GET /seances/new
+  def new
+    @seance = Seance.new
+    @halls = Hall.all
+    @movies = Movie.all
+  end
+  # POST /seances or /seances.json
+  def create
+    @seance = Seance.new(seance_params)
```
Comment (baniak66) [inline app/controllers/seances_controller.rb:14]: You don't have a variable or method `seance_params`. Have you run this code? 😉

File: app/controllers/seances_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    respond_to do |format|
+      if @seance.save
+        format.html { redirect_to seance_url(@seance), notice: "Seance was successfully created." }
+        format.json { render :show, status: :created, location: @seance }
```
Comment (baniak66) [inline app/controllers/seances_controller.rb]: Do you need a response in JSON format?

File: db/migrate/20220913071239_rename_seances_rows.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class RenameSeancesRows < ActiveRecord::Migration[7.0]
```
Comment (baniak66) [inline db/migrate/20220913071239_rename_seances_rows.rb]: Within one PR you could rename those columns in the CreateSeances migration. Just rollback, rename columns and migrate again.

File: app/controllers/seances_controller.rb (hunk 1 above)
Comment (meef3k) [reply app/controllers/seances_controller.rb:14]: I checked it after PR and it didn't work so I added `seance_params` so now it works

File: app/controllers/seances_controller.rb (hunk 2 above)
Comment (meef3k) [reply app/controllers/seances_controller.rb]: I don't need it now, so I will delete it. Thanks :D

**Review by baniak66 — APPROVED** [review]
LGTM 👍

**Pull request merged**